roles/lib_utils/src/class/repoquery.py

```diff
@@ -32,6 +35,9 @@ def build_cmd(self):
         if self.show_duplicates:
             repo_cmd.append('--show-duplicates')
 
+        if self.ignore_excluders:
+            repo_cmd.append('--config=' + self.tmp_file.name)
```

Comment:
Is there any difference between `'--config={}'.format("ddd")` and `"--config=%s" % "ddd"`?